ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -236,7 +236,7 @@ export const useNewMhrRegistration = () => {
     const draftFilings = await getMhrDrafts()
     const myMhrHistory = await mhrRegistrationHistory(true, sortOptions)
     const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings, sortOptions)
-    setMhrTableHistory(filteredMhrHistory)
+    setMhrTableHistory(myMhrHistory)
```

Comment:
is `filteredMhrHistory` still used? 
How does it handle the draft filings now?

Reply:
Good questions.. I am wondering the same thing..

Reply:
Working on a fix for dealing with drafts, I missed this